### PR TITLE
added utterances

### PR DESCRIPTION
**Pull request by ahmedelq**
## Website
https://utteranc.es/
## Examples
https://haxe.org/manual/introduction.html
https://mazhuang.org/2020/10/28/add-vscode-to-right-click/
https://dmitripavlutin.com/react-context-and-usecontext/

### CODE REVIEW

File: src/technologies/u.json [hunk 1]
```diff
@@ -194,6 +194,16 @@
     "url": "/cgi-bin/UCEditor\\?",
     "website": "http://ultracart.com"
   },
+  "utterances": {
+    "description":"A lightweight comments widget built on GitHub issues",
+    "cats": [
+      15
+    ],
+    "dom": "iframe[src*='^(https?:\/\/)?(www\.)?utteranc\.es/client.js']",
```
Comment (nurbek91) [inline src/technologies/u.json:202]: "dom": "iframe[src*='^(https?:\/\/)?(www\.)?utteranc\.es/client.js']",

"dom": "iframe[src*='utteranc.es/client.js']",

File: src/technologies/u.json [hunk 2]
```diff
@@ -194,6 +194,16 @@
     "url": "/cgi-bin/UCEditor\\?",
     "website": "http://ultracart.com"
   },
+  "utterances": {
+    "description":"A lightweight comments widget built on GitHub issues",
```
Comment (nurbek91) [inline src/technologies/u.json:198]: utterances is a lightweight comments widget built on GitHub issues.

File: src/technologies/u.json [hunk 3]
```diff
@@ -194,6 +194,16 @@
...
+    "cats": [
+      15
+    ],
+    "dom": "iframe[src*='^(https?:\/\/)?(www\.)?utteranc\.es/client.js']",
+    "xhr": "^(https?:\/\/)?(www\.)?utteranc\.es\/.*\.(html|js|css)"
+    "icon": "utterances.png",
```
Comment (nurbek91) [inline src/technologies/u.json:204]: icon should be 32x32
also we can add oss property

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for PR!!!
A few tips to perfect your request

File: src/technologies/u.json (hunk 3 above)
Comment (ahmedelq) [reply src/technologies/u.json:204]: done

File: src/technologies/u.json [hunk 4]
```diff
@@ -194,6 +194,17 @@
     "url": "/cgi-bin/UCEditor\\?",
     "website": "http://ultracart.com"
   },
+  "utterances": {
+    "description":"Utterances is a lightweight comments widget built on GitHub issues",
+    "cats": [
+      15
+    ],
+    "dom": "iframe[src*='utteranc.es/client.js']",
+    "xhr": "\.utteranc\.es",
```
Comment (nurbek91) [inline src/technologies/u.json:203]: `\\.utteranc\\.es`

File: src/technologies/u.json [hunk 5]
```diff
@@ -194,6 +194,17 @@
     "url": "/cgi-bin/UCEditor\\?",
     "website": "http://ultracart.com"
   },
+  "utterances": {
+    "description":"Utterances is a lightweight comments widget built on GitHub issues",
```
Comment (nurbek91) [inline src/technologies/u.json:198]: utterances is a lightweight comments widget built on GitHub issues.

**Comment by AliasIO** [comment]
I fixed those up, thanks!

**Pull request merged**